dev-games/ogre/ogre-1.10.12.ebuild

```diff
@@ -13,41 +15,46 @@ LICENSE="MIT public-domain"
 SLOT="0/1.10.0"
 KEYWORDS="~amd64 ~arm ~x86"
 
-IUSE="cg doc double-precision examples +freeimage gl3plus gles2 gles3 ois +opengl profile tools"
+IUSE="beta-components +cache cg doc double-precision egl examples +freeimage
+	gles2 ois +opengl profile +resman-legacy resman-pedantic resman-strict
+	tools"
 
-REQUIRED_USE="examples? ( ois )
-	gles3? ( gles2 )
-	gl3plus? ( opengl )"
+REQUIRED_USE="
+	|| ( gles2 opengl )
+	^^ ( resman-legacy resman-pedantic resman-strict )
+	examples? (
+		beta-components
+		ois
+	)"
 
 RESTRICT="test" #139905
 
 RDEPEND="
 	dev-libs/boost
 	dev-libs/zziplib
 	media-libs/freetype:2
-	virtual/glu
-	virtual/opengl
 	x11-libs/libX11
 	x11-libs/libXaw
 	x11-libs/libXrandr
 	x11-libs/libXt
 	cg? ( media-gfx/nvidia-cg-toolkit )
+	egl? ( media-libs/mesa[egl] )
 	freeimage? ( media-libs/freeimage )
-	gl3plus? ( >=media-libs/mesa-9.2.5 )
-	gles2? ( >=media-libs/mesa-9.0.0[gles2] )
-	gles3? ( >=media-libs/mesa-10.0.0[gles2] )
+	gles2? ( media-libs/mesa[gles2] )
 	ois? ( dev-games/ois )
-	tools? ( dev-libs/tinyxml[stl] )"
```

Comment:
what happened to the tinyxml dependency? is it now switched to bundling it again? we should leave at least a `TODO` note around, and also file a bug against the `bundled-lib` tracker, until a build system patch is fixing this again.

Reply:
Actually tinyxml is not meant to be linked as a library. It is meant to be put into and compiled with your project.
Further more I removed this, because the build system simply ignores any installed tinyxml. That line was deceptive.

well, TinyXML 1 is legacy. Projects really should use TinyXML 2, which only consist of tinyxml2.cpp and tinyxml2.h. (Even less sense in making it an .so... That is 84K large. Or, overly huge in that case.)

To tell you the truth, I am not very happy with Gentoo building tinyxml[2].so files. If they must be precompiled, despite their minuscule size, than static archives make much more sense.

However, maybe I can "convince" the Ogre build system to not ignore pre-installed TinyXML versions. And maybe, while I am at it, I can teach it to use TinyXML2.

Reply:
Some more information:

Although the bundled tinyxml bits are only are only used `if(NOT TINYXML_FOUND)`, the actual looking for a system installed tinyxml have been removed from the build system. `TINYXML_FOUND` is always false.

Reply:
That makes sense to me then.

Reply:
Well, I have good news. Luckily the searching and including of a system wide installed tinyxml instance is not as complicated as I first thought. I am currently testing a patch that "unbundles" tinyxml again.

I *think* that upstream removed their FindTinyXML.cmake stuff, because they explicitly need STL support, and got too many bug reports about things breaking when people tried without.

Okay, that's pure speculation. And irrelevant as we can demand the configuration we need anyway.

Reply:
Oddly enough the TinyXML detection is still there in 2.1. :-)

Reply:
Oddly enough the TinyXML detection is still there in 2.1. :-)